apps/web/vibes/soul/primitives/products-carousel/index.tsx

```diff
@@ -102,26 +103,28 @@ export function ProductsCarousel({
 export function ProductsCarouselSkeleton({
   className,
   placeholderCount = 8,
-  pending = false,
 }: {
   className?: string;
   placeholderCount?: number;
-  pending?: boolean;
 }) {
   return (
-    <Carousel className={className} data-pending={pending ? '' : undefined}>
```

Comment:
Don't we want to preserve the ability to pass in `pending` so that we can distinguish from a pending skeleton from a skeleton we use on an empty state (which isn't pending)?